doit/cmd_completion.py

```diff
@@ -98,8 +97,12 @@ def _generate_bash(self, opt_values, pos_args):
 
         # if hardcode tasks
         if opt_values['hardcode_tasks']:
-            self.task_list, _ = self.loader.load_tasks(
-                self, opt_values, pos_args)
+            if isinstance(self.loader, TaskLoader2):
+                self.loader.setup(opt_values)
+                self.task_list = self.loader.load_tasks(cmd=self, pos_args=pos_args)
```

Comment:
Uhhmm. I have not thought about the possibility of calling ``load_tasks`` without (or before) calling `` load_doit_config``.

Not sure about it. I think this should be forbidden somehow. ``load_doit_config`` might perform some configuration changes in the loader itself... What do you think?

Reply:
I can add the config load call here as well. I would not go so far as to double-checking in the loader classes, I did not see anything like this done elsewhere.

Reply:
yes. double-checking might be overkill, but should be noted somewhere...

Reply:
Ok, I'll add to docs as well.